build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -12,14 +12,6 @@ featureGates:
 # promoted to GA in v1.14.
 {{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "AntreaProxy" "default" true) }}
 
-# Enable TopologyAwareHints in AntreaProxy. This requires AntreaProxy and EndpointSlice to be
-# enabled, otherwise this flag will not take effect.
-{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "TopologyAwareHints" "default" true) }}
-
-# Enable ServiceTrafficDistribution in AntreaProxy. This requires AntreaProxy and EndpointSlice to be
-# enabled, otherwise this flag will not take effect.
-{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "ServiceTrafficDistribution" "default" true) }}
-
```

Comment:
should we do the same thing that we did for `AntreaProxy` FeatureGate above? Probably worth looking into the PR history for some context